packages/vm/testdata/hackatom.wasm

```diff
@@ -1 +1 @@
-hackatom_0.14.wasm
\ No newline at end of file
+hackatom_0.13.wasm
```

Comment:
This link should not be changed. The contract goes into `hackatom_0.14.wasm`, see the build command here: https://github.com/CosmWasm/cosmwasm/blob/master/packages/vm/README.md#setup

hackatom_0.13.wasm was skipped. We only have hackatom_0.12.wasm and hackatom_0.14.wasm

Reply:
Fixed, thanks.

Reply:
> https://github.com/CosmWasm/cosmwasm/blob/master/packages/vm/README.md#setup

By the way, is the reference to the stacking contract above still valid?

Reply:
> is the reference to the stacking contract above still valid?

Nope, good catch. We'll soon add an IBC testing contract to `packages/vm`. Then we can update the text.